src/components/Containers/Header/Header.js

```diff
@@ -53,7 +57,8 @@ export const Header = () => {
       </div>
 
       <div className={utils.object.toClasses(styles.right, 'row')}>
-        <Tab label={TAB_DISCORD_TXT} onClick={onTabDiscordClick}></Tab>
+        <Tab label={TAB_FAQ} onClick={onTabFaqClick}></Tab>
+        <Tab label={TAB_DISCORD_TXT} style={{marginRight: '5px'}} onClick={onTabDiscordClick}></Tab>
```

Comment:
Remove inline css styles and manage it in `Tab` component

Reply:
You can handle it using css by `last-child` and `first-child` selectors.
Read here:
https://www.w3schools.com/cssref/sel_firstchild.asp
https://www.w3schools.com/cssref/sel_last-child.asp

Reply:
done